jme3-terrain/src/main/java/com/jme3/terrain/geomipmap/TerrainPatch.java

```diff
@@ -798,19 +801,22 @@ protected void setLodBottom(int lodBottom) {
 
     @Override
     public int collideWith(Collidable other, CollisionResults results) throws UnsupportedCollisionException {
-        if (refreshFlags != 0)
-            throw new IllegalStateException("Scene graph must be updated" +
-                                            " before checking collision");
-
-        if (other instanceof BoundingVolume)
-            if (!getWorldBound().intersects((BoundingVolume)other))
+        if (refreshFlags != 0) {

```

Comment:
I think only `RF_BOUND` and `RF_TRANSFORM` should be considered similar to what Geometry does.

https://github.com/jMonkeyEngine/jmonkeyengine/blob/375015f14c5f3cbda18302b3aa990e5f023e9da1/jme3-core/src/main/java/com/jme3/scene/Geometry.java#L465

so it should be

```
if ((refreshFlags & (RF_BOUND | RF_TRANSFORM)) != 0) {
....
}
```